### ISSUE TITLE
Only one operation can be active at a time / How to stop or save the task?

### ISSUE THREAD

**Issue by ereyesc95** [jamesmontemagno/MediaPlugin#820]
## **ONLY Post issues against Latest Beta!!!**

Make sure you follow all of the setup directions before posting an issue: https://github.com/jamesmontemagno/MediaPlugin#important-permission-information

If you are creating an issue for a BUG please fill out this information. If you are asking a question or requesting a feature you can delete the sections below. 

**Failure to fill out this information will result in this issue being closed.** If you post a full stack trace in a bug it will be closed, please post it to http://gist.github.com and then post the link here.

## Bug Information

Version Number of Plugin: 4.0.1.5
Device Tested On: Motorola Moto G
Simulator Tested On: -
Version of VS: 15.8.4
Version of Xamarin: 4.11.0.776
Versions of Xamarin iOS: 12.0.0.15

### Steps to reproduce the Behavior
Open the gallery, then go to the phone's menu, then open the app again and try to access to gallery again
### Expected Behavior
Open Gallery
### Actual Behavior
App Crashes, Error: Only one operation can be active at a time
### Code snippet
var file = await CrossMedia.Current.PickPhotoAsync(new PickMediaOptions()
            {
                PhotoSize = Plugin.Media.Abstractions.PhotoSize.Large
            });
### Screenshots

When I enter to the gallery, then go to the phone's menu and select my app again I tap to access the gallery again and I get the message: Only one operation can be active at a time. What I want to do is once I go back to my app make sure to finish any started tasks before starting a new one. How to? Thanks.

**Comment by DuncWatts** [comment]
Duplicate of #144 which has been open for over 3 years, not sure this one will ever be fixed.
